modules/configuring-queue-proxy.adoc

```diff
@@ -0,0 +1,68 @@
+:_content-type: PROCEDURE
+[id="serverless-configuring-queue-proxy_{context}"]
+= Configuring Queue Proxy Resources for a Knative Service
+
+Apart from configuring the Queue Proxy resource requests and limits globally in the deployment configmap, you can set them at the service level using the corresponding annotations targeting CPU, memory, and ephemeral storage resource types.
+
+.Prerequisites
+
+* {pipelines-title} must be installed on your cluster.
+
+* You have installed the OpenShift (`oc`) CLI.
+
+* You have installed the Knative (`kn`) CLI.
```

Comment:
I would say none of the prerequisites above are required. Certainly not "Pipelines". 
I'd consider them required if we provided an example how to apply a yaml file (then `oc` would be required), or if we provided an example using `kn`. None of these are used below. We just show the service yaml.